### ISSUE TITLE
False positive of `IndentationStyle` with tabs and alignment

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#7884]
Hello.

I'm trying to update RuboCop to version 0.82.0 for [Flame](https://github.com/AlexWayfer/flame).

I'm using tabs in this project.

And there is alignment in `.gemspec` file:

```ruby
	s.description = 'Use controller\'s classes with instance methods' \
	                ' as routing actions, mounting its in application class.'
```

--------

## Expected behavior

No offenses.

## Actual behavior

There is offense about spaces.

## Steps to reproduce the problem

Code placed above.

## RuboCop version

```
0.82.0 (using Parser 2.7.1.1, running on ruby 2.7.1 x86_64-linux)
```

**Comment by AlexWayfer** [comment]
Forgot to /cc @DracoAter

**Comment by DracoAter** [comment]
Currently the cop (when EnforcedStyle: tabs) looks for spaces before any alpha-numeric in a line, expecting that there should be only indentation. But turns out there can be alignment too.

Probably we could switch the behaviour in such case to look for expected number of tabs in the beginning of the line?

**Comment by AlexWayfer** [comment]
> Probably we could switch the behaviour in such case to look for expected number of tabs in the beginning of the line?

Ideally, we should check a number of tabs (as indentation) and a number of spaces (alignment, relatively to the previous line). At least number of tabs relatively to the start of previous line (and left alignment to developer responsibility).

**Comment by AlexWayfer** [comment]
I guess here should be a ~bug label.

**Comment by AlexWayfer** [comment]
It's probably not too popular case, and heredocs make good job, so it can be closed probably according to RuboCop policy (if I've understood and remember it correctly).
